src/dmd/dsymbolsem.d

```diff
@@ -6840,7 +6832,20 @@ bool determineFields(AggregateDeclaration ad)
             return 1;
 
         if (v.aliassym)
-            return 0;   // If this variable was really a tuple, skip it.
+        {
+            // If this variable was really a tuple, process each element.
+            if (auto tup = v.aliassym.isTupleDeclaration())
+            {
+                foreach (o; *tup.objects)
+                {
+                    if (auto e = o.isExpression())
+                        if (auto se = e.isDsymbolExp())
```

Comment:
Perhaps have a `foreachDsymbolExp` helper to wrap this loop into a delegate callback?

Reply:
Implemented in #14247

Reply:
PR merged